
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/try_catch_same_type.res[0m:[2m2:8-14[0m

  1 [2mâ”‚[0m let x = try {1} catch {
  [1;31m2[0m [2mâ”‚[0m | _ => [1;31m"hello"[0m
  3 [2mâ”‚[0m }
  4 [2mâ”‚[0m 

  This has type: [1;31mstring[0m
  But this try/catch is expected to return: [1;33mint[0m

  The [1;33mtry[0m body and the [1;33mcatch[0m block must return the same type.
  To fix this, change your try/catch blocks to return the expected type.
  
  You can convert [1;33mstring[0m to [1;33mint[0m with [1;33mInt.fromString[0m.